docs/esa/xmm_newton.rst

```diff
@@ -133,7 +133,30 @@ This will show the column details of the table 'v_all_observations' in XSA TAP s
   >>> XMMNewton.get_epic_images('tarfile.tar', band=[1,2], instrument=['M1'])
   {1: {'M1': '/home/dev/esa/0405320501/pps/P0405320501M1S002IMAGE_1000.FTZ'}, 2: {'M1': '/home/dev/esa/0405320501/pps/P0405320501M1S002IMAGE_2000.FTZ'}}
 
-This will extract the European Photon Imaging Camera (EPIC) images within the specified TAR file, bands, and instruments. It will also return a dictionary containing the paths to the extracted files.        
+This will extract the European Photon Imaging Camera (EPIC) images within the specified TAR file, bands, and instruments. It will also return a dictionary containing the paths to the extracted files.
+
+------------------------------------------------------------------------------
+7. Getting the European Photon Imaging Camera (EPIC) metadata from the XSA TAP 
```

Comment:
it's perfectly fine to number manually, but you could also consider leaving it to the rest rendering